ios/RCTWebRTC/WebRTCModule+Permissions.m

```diff
@@ -29,27 +31,28 @@ - (AVMediaType)avMediaType:(NSString *)mediaType {
         reject(@"invalid_type", @"Invalid media type", nil);
         return;
     }
+        AVAuthorizationStatus status = [AVCaptureDevice authorizationStatusForMediaType:mediaType_];
```

Comment:
Why is this code re-indented? Please run `npm run format` which uses clang-format.